.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         # remove until I sort out CI issues for truffle
         # truffleruby,
         # truffleruby-head,
-        ruby: [2.3, 2.4, 2.5, 2.6, 2.7, jruby, jruby-head]
+        ruby: [2.4, 2.5, 2.6, 2.7, 3.0, jruby, jruby-head]
```

Comment:
Is there a reason to drop support for Ruby 2.3? Are there any failing tests?

Reply:
I am not sure why I did that honestly. Opened #417 to add it back.